app/src/main/java/com/nilhcem/blenamebadge/adapter/DrawableAdapter.kt

```diff
@@ -41,8 +41,12 @@ class DrawableAdapter(private val context: Context, private val list: List<Drawa
         }
     }
 
-    fun getSelectedItem(): Any {
-        return if (selectedPosition == -1) -1 else list[selectedPosition]
+    fun getSelectedItem(): DrawableInfo? {
+        return if (selectedPosition == -1) null else list[selectedPosition]
```

Comment:
Return default item in case of -1

Reply:
@iamareebjamal if default item is returned, the preview will display it along with animation modes/speeds set by the user. But if preview is clicked without any parameter, the default item is to be shown with no animation, in fixed mode. I will return null and then check if the returned value is null, and then preview the default item with no animation. Is it alright that way?

Reply:
I have made the changes. Please tell if this way is alright?